### ISSUE TITLE
Team Split Test Bug

### ISSUE THREAD

**Issue by jdinan** [Sandia-OpenSHMEM/SOS#972]
Looks like the compiler is right about this. Should we add error checks or initialize the dims to 1?
```
../../../test/spec-example/shmem_team_split_2D.c: In function ‘main’:
../../../test/spec-example/shmem_team_split_2D.c:56:3: warning: ‘zdim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   56 |   for (int zdx = 0; zdx < zdim; zdx++) {
      |   ^~~
../../../test/spec-example/shmem_team_split_2D.c:57:5: warning: ‘ydim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   57 |     for (int ydx = 0; ydx < ydim; ydx++) {
      |     ^~~
../../../test/spec-example/shmem_team_split_2D.c:58:7: warning: ‘xdim’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   58 |       for (int xdx = 0; xdx < xdim; xdx++) {
      |       ^~~
```

**Comment by davidozog** [comment]
Indeed, I think so.  Could it be a spec doc-edit, or should I prepare a reading?  You probably recall we're trying to keep these `spec-example` tests verbatim with what's upstream.

**Comment by jdinan** [comment]
Let's post it as a PR against the spec. Do you think it will cause testing failures? If so, I would also apply the same fix here.

**Comment by davidozog** [comment]
I think it's unlikely this test will fail because of this (something would have to go awry with `shmem_n_pes` or `cbrt(npes)`).  

However, it looks like @rdesai16 actually resolved the compiler warning in her v1.5.0rc PR (#955) (I somehow missed it during review).  I think it's ok for us to make the change in SOS, and this issue will track the upstream PR as well.

Any concerns with that?
